cognite/client/_api/simulators/integrations.py

```diff
@@ -42,34 +41,48 @@ def __iter__(self) -> Iterator[SimulatorIntegration]:
 
     @overload
     def __call__(
-        self, chunk_size: int, filter: SimulatorIntegrationFilter | None = None, limit: int | None = None
+        self,
+        chunk_size: int,
+        limit: int | None = None,
+        simulator_external_ids: str | SequenceNotStr[str] | None = None,
+        active: bool | None = None,
```

Comment:
change the limit to be the last paramater on all of these.
same as other APIs do it

I think @doctrino will also agree

Reply:
I see only data models don't follow this order though, lets ask him first